HOWTO.rst

```diff
@@ -111,3 +111,57 @@ Different flavours of the documentation can be generated with wildcard entries:
     make sphinx.html # Builds HTML doc at docs/build/html/index.html
     make sphinx.epub # Builds EPUB doc at docs/build/epub
 
+----------
+Windows Troublshoot
+----------
+1. Download 7-zip from https://www.7-zip.org/download.html (to extract .zip/.tgz/.tar)
+2. Download Apache Spark from http://spark.apache.org/downloads.html (Spark Release 3.1.2, Pre-built for Hadoop 3.2 and later),
```

Comment:
I propose that you do not specify the version here. Because, if Spark 3.4 is released next year, you will need to update the version. Instead "Download the latest version of Apache Spark from ..." should be sufficient.